.github/workflows/build.yml

```diff
@@ -33,7 +36,13 @@ jobs:
     - name: cmake
       run: |
         cd build
-        CYMBL=OFF cmake ../src/llvm -GNinja -DLLVM_ENABLE_PROJECTS="llvm;clang;mlir" -DCMAKE_BUILD_TYPE=${{ matrix.build }} -DCMAKE_C_COMPILER=/bin/clang -DCMAKE_CXX_COMPILER=/bin/clang++ -DCMAKE_ASM_COMPILER=/bin/clang -DCMAKE_CXX_FLAGS="-Wno-c++11-narrowing"
+        CYMBL=OFF cmake ../src/llvm -GNinja \
+        -DLLVM_ENABLE_PROJECTS="llvm;clang;mlir" \
```

Comment:
Restoring this back to the cymbl clang should restore the cache and parallelism

Reply:
I see. Will do that. Thanks!